internal/frontend/http/pxe.go

```diff
@@ -92,6 +80,20 @@ func (f *Frontend) handlePXE(ctx context.Context, w http.ResponseWriter, _ *http
 		return err
 	}
 
+	if prof.SecureBootEnabled() {
+		return ensure.Value(template.New("secureboot.ipxe").
+			Parse(securebootIPXE)).
+			Execute(w,
+				struct {
+					UKIURL  string
+					Cmdline string
+				}{
+					UKIURL:  f.options.ExternalPXEURL.JoinPath("image", schematicID, versionTag, fmt.Sprintf("%s-%s-secureboot-uki.efi", prof.Platform, prof.Arch)).String(),
+					Cmdline: string(cmdline),
```

Comment:
I'm a bit confused about this one - if SecureBoot is enabled already, UKI should ignore the cmdline passed?

Reply:
Adding cmdline won't hurt, but still not sure I fully understand the flow

Reply:
well with pxe it just boots the uki as normal EFI PE executable, UKI command line is parsed if using sd-boot/sd-stub combo, pxe bypasses that